tutorial/source/index.rst

```diff
@@ -3,92 +3,202 @@
+:doc:`the guide to implementing custom variational objectives <custom_objectives>`,
+and a companion example that walks through :doc:`implementing "Boosting BBVI" <boosting_bbvi>`.
+
+Particularly enthusiastic users and potential contributors, especially those interested in contributing to Pyro's core components,
+may even be interested in how Pyro itself works under the hood, partially described in the series :ref:`understanding-pyros-internals`.
+The :doc:`mini-pyro example <minipyro>` contains a complete and heavily commented implementation of a small version of the Pyro language in just a few hundred lines of code,
+and should serve as a more digestable introduction to the real thing.
+
+Tools for specific problems
+-----------------------------
+Pyro is a mature piece of open-source software, not a just science project,
+and in addition to the core machinery for modelling and inference,
+it includes quite a bit of dedicated domain- or problem-specific modelling functionality.
+
+One particular area of strength is time-series modelling via `pyro.contrib.forecasting <http://docs.pyro.ai/en/dev/contrib.forecast.html>`_,
+a library for scaling hierarchical, fully Bayesian models of multivariate time series to thousands or millions of series and datapoints.
+This is described in the series :ref:`time-series`.
+
+Another area of strength is probabilistic machine learning with Gaussian processes.
+`pyro.contrib.gp <http://docs.pyro.ai/en/dev/contrib.gp.html>`_, described in the series :ref:`gaussian-processes`,
+is a library within Pyro implementing a variety of exact or approximate Gaussian process models compatible with Pyro's inference engines.
+Pyro is also fully compatible with `GPyTorch <https://gpytorch.ai/>`_, a dedicated library for scalable GPs,
+as described in `their Pyro example series <https://github.com/cornellius-gp/gpytorch/tree/master/examples/07_Pyro_Integration>`_.
+
+List of Tutorials
+==================
 
 .. toctree::
-   :maxdepth: 2
-   :caption: Introduction:
+   :maxdepth: 1
+   :caption: Introductory Tutorials
+   :name: introductory-tutorials
 
    intro_part_i
    intro_part_ii
    svi_part_i
    svi_part_ii
    svi_part_iii
-   tensor_shapes
-   mle_map
 
 .. toctree::
-   :maxdepth: 2
-   :caption: Advanced:
+   :maxdepth: 1
+   :caption: Hands On with Pyro and PyTorch
+   :name: hands-on-with-pyro-and-pytorch
 
-   enumeration
-   custom_objectives
-   jit
-   minipyro
-   effect_handlers
+   bayesian_regression
+   bayesian_regression_ii
+   tensor_shapes
    modules
-   normalizing_flows_i
-   contrib_funsor_intro_i
-   contrib_funsor_intro_ii
+   jit
 
 .. toctree::
-   :maxdepth: 2
-   :caption: Examples:
+   :maxdepth: 1
+   :caption: Deep Generative Models
+   :name: deep-generative-models
 
    vae
-   bayesian_regression
-   bayesian_regression_ii
-   dmm
-   air
    ss-vae
-   stable
    cvae
+   normalizing_flows_i
+   dmm
+   air
+   cevae
+   sparse_gamma
+   svi_horovod
```

Comment:
The Horovod example seems more like an "advanced PyTorch" tutorial? I don't even remember what the model is, only that it shows how to perform distributed training.